app/Commands/LocalDocker/Bootstrap.php

```diff
@@ -0,0 +1,155 @@
+<?php declare( strict_types=1 );
+
+namespace App\Commands\LocalDocker;
+
+use App\Commands\Open;
+use App\Services\Docker\Local\Config;
+use App\Services\ProjectBootstrapper;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\Validator;
+
+/**
+ * Class Bootstrap
+ *
+ * @package App\Commands\LocalDocker
+ */
+class Bootstrap extends BaseLocalDocker {
+
+    /**
+     * The signature of the command.
+     *
+     * @var string
+     */
+    protected $signature = 'bootstrap';
+
+    /**
+     * The description of the command.
+     *
+     * @var string
+     */
+    protected $description = 'Bootstrap WordPress: Install core, create an admin user';
+
+    /**
+     * Execute the console command.
+     *
+     * @param  \App\Services\Docker\Local\Config  $config
+     *
+     * @param  \App\Services\ProjectBootstrapper  $bootstrapper
+     *
+     * @return int
+     */
+    public function handle( Config $config, ProjectBootstrapper $bootstrapper ) {
+        $this->info( 'Alright, let\'s get ready to configure WordPress!' );
+
+        $email                = $this->ask( 'Enter your email address' );
+        $username             = $this->ask( 'Enter your admin username' );
+        $password             = $this->secret( 'Enter your password' );
+        $passwordConfirmation = $this->secret( 'Confirm your password' );
+
+        $validator = Validator::make( [
+            'email'                 => $email,
+            'username'              => $username,
+            'password'              => $password,
+            'password_confirmation' => $passwordConfirmation,
+        ], [
+            'email'                 => [ 'required', 'email' ],
+            'username'              => [ 'required' ],
+            'password'              => [ 'required', 'same:password_confirmation' ],
+            'password_confirmation' => [ 'required' ],
+        ], [
+            'required' => 'The :attribute field is required',
+            'same'     => 'The :attribute and :other must match',
+            'email'    => 'Invalid email address',
+        ] );
+
+        if ( $validator->fails() ) {
+
+            foreach ( $validator->errors()->all() as $error ) {
+                $this->error( $error );
+            }
+
+            return self::EXIT_ERROR;
+        }
+
+        $this->task( 'Starting docker containers', call_user_func( [ $this, 'startContainers' ] ) );
+        $this->task( 'Bootstrapping project', call_user_func( [ $this, 'bootstrap' ], $config, $bootstrapper ) );
+        $this->task( 'Installing WordPress', call_user_func( [ $this, 'installWordpress' ], $config, $email, $username, $password ) );
+
+        $bootstrapper->restoreObjectCache( $config->getProjectRoot() );
```

Comment:
Minor code structure quibble: it's odd that the object cache is disabled in the `bootstrap()` method but then restored here. The lack of symmetry makes it hard to track down what's happening. Instead, could this be:

```
$bootstrapper->renameObjectCache( $config->getProjectRoot() );

$this->task( 'Starting docker containers', call_user_func( [ $this, 'startContainers' ] ) );
$this->task( 'Bootstrapping project', call_user_func( [ $this, 'bootstrap' ], $config, $bootstrapper ) );
$this->task( 'Installing WordPress', call_user_func( [ $this, 'installWordpress' ], $config, $email, $username, $password ) );

$bootstrapper->restoreObjectCache( $config->getProjectRoot() );
```